app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getAllTopics } = require("./controller/topics.controller");
+const { handleCustomErrors } = require("./errors");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/topics", getAllTopics);
+
+app.get("/api/nvnewbs", getAllTopics);
+
```

Comment:
You should also include middleware that would capture any unexpected i.e. 500 errors

Reply:
You've done this ✅